docs/src/conversions.md

```diff
@@ -6,6 +6,42 @@ DocTestSetup = quote
 end
 ```
 
+## Converting to plain Dates types without timezone information.
+
+To convert from a `ZonedDateTime` into a vanilla `DateTime`, one can use the `DateTime` constructor.
+Passing either `Local` to directly drop the timezone,  or `UTC` to convert to UTC time first.
```

Comment:
I'm hesitant to use the word "convert" as `UTC` is just returning a stored value where `Local` is where the conversion occurs:
```suggestion
Passing either `Local` to directly drop the time zone,  or `UTC` to extract the time as UTC instead.
```